### ISSUE TITLE
Wrong OpenGL equivalent for TEX_FORMAT_BC1_UNORM_SRGB

### ISSUE THREAD

**Issue by code-monkey-101** [DiligentGraphics/DiligentCore#380]
In GraphicTypes.h, GL_COMPRESSED_SRGB_S3TC_DXT1_EXT is mentioned as the OpenGL counterpart of TEX_FORMAT_BC1_UNORM_SRGB (DXGI_FORMAT_BC1_UNORM_SRGB).

I think, the correct counterpart is GL_COMPRESSED_SRGB_ALPHA_S3TC_DXT1_EXT.

**Comment by TheMostDiligent** [comment]
The engine currently uses `GL_COMPRESSED_SRGB_S3TC_DXT1_EXT` for `TEX_FORMAT_BC1_UNORM_SRGB`.

**Comment by code-monkey-101** [comment]
That will probably ignore the alpha bit.

**Comment by TheMostDiligent** [comment]
Do you need alpha?

**Comment by code-monkey-101** [comment]
No, I'm good, thanks. I just noticed this when I was googling for the right OpenGL equivalent for BC1_UNORM_SRGB for my own engine ;)
